diff --git a/node_modules/formik/dist/withFormik.d.ts b/node_modules/formik/dist/withFormik.d.ts
index dcc9f52..d00d1d9 100644
--- a/node_modules/formik/dist/withFormik.d.ts
+++ b/node_modules/formik/dist/withFormik.d.ts
@@ -55,7 +55,7 @@ export interface WithFormikConfig<Props, Values extends FormikValues = FormikVal
      */
     validate?: (values: Values, props: Props) => void | object | Promise<any>;
 }
-export declare type CompositeComponent<P> = React.ComponentClass<P> | React.StatelessComponent<P>;
+export declare type CompositeComponent<P> = React.ComponentClass<P> | React.FunctionComponent<P>;
 export interface ComponentDecorator<TOwnProps, TMergedProps> {
     (component: CompositeComponent<TMergedProps>): React.ComponentType<TOwnProps>;
 }
